server/database/data/rawdaty.sql

```diff
@@ -0,0 +1,50 @@
+BEGIN;
+
+DROP TABLE IF EXISTS kindergarten,user, location, comments CASCADE;
+
+CREATE TABLE kindergarten(
+id SERIAL PRIMARY KEY,
+kindergarten_name VARCHAR(50)	NOT NULL,
+user_id	SERIAL	REFERENCES user(id),
+cover_image	TEXT NOT NULL,
+description	TEXT NOT NULL,
+location_id	SERIAL	REFERENCES location(id),
+phone_number INTEGER NOT NULL,
+latitude INTEGER,
+longitude INTEGER,
+min_price INTEGER,	
+max_price INTEGER,	
+periods	TEXT,	
+images_gallery TEXT,	
+request_status TEXT,	
+is_enable BOOLEAN SET DEFAULT false,
```

Comment:
This BOOLEAN data type accepts `0&1` only so you should know that before start coding, in my opinion, I think you might use `VARCHAR(50)` and store it as a `true`, `false` string